src/test/javascript/spec/component/admin/user-management.component.spec.ts

```diff
@@ -44,31 +71,28 @@ describe('UserManagementComponent', () => {
     // The admin module is lazy loaded - we therefore need a dummy test to load
```

Comment:
Can you verify whether this statement still holds?

We started to remove these tests because to our knowledge the component can never be undefined (btw undefined not null!).
Here is an explicit comment but I can't see any "admin module" or where it's verified.
Or if it refers to the test admin module, it's removed now so....

Reply:
true, it makes no sense. I removed it